inverse_modulo(prime).cpp

```diff
@@ -0,0 +1,38 @@
+  
```

Comment:
Please add time complexity here(at the top of file, after info on the algo) as well